### ISSUE TITLE
ChatGPT GitHub Action for gene-associated modification

### ISSUE THREAD

**Issue by feiranl** [SysBioChalmers/Human-GEM#483]
### Description of the issue:
We are thinking that we can make ChatGPT as  github action so that it can double check the modification made in the pull request.

1) Go to Uniprot extract the annotation for the `modified gene`
2) check wether the gene is associated with the subsystem and the substrates
3) according to the output, then define pass/fail for the model modification

**Comment by mihai-sysbio** [comment]
Some more clarifications would be appreciated:
- What is a `modified` gene? A gene entry in the yaml that was added in that PR?
- What is implied by "go to UniProt"?
- How does one check whether a gene is associated with the system and substrates? Just ask ChatGTP?

**Comment by mihai-sysbio** [comment]
Here is a random example:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/23480589/233341164-5e8dfdb2-68e8-4870-bb93-59f950565604.png">

**Comment by mihai-sysbio** [comment]
And here is the Bing answer to the same question:

<img width="747" alt="image" src="https://user-images.githubusercontent.com/23480589/233345398-82b8b217-54c6-4a19-b4d6-2b5a9b9cfd57.png">

**Comment by feiranl** [comment]
@guozhetao Could you handle this task?

**Comment by mihai-sysbio** [comment]
In my previous tests with publicly-available AI models, the responses I would get were not trustworthy. I'd be curious to see how this changed since, and perhaps if the question can be formulated differently for a more accurate answer.

**Comment by feiranl** [comment]
Yes, it seems much better right now with the development of LLMs. Will update soon.
